app/lib/LevelLoader.coffee

```diff
@@ -92,12 +92,20 @@ module.exports = class LevelLoader extends CocoClass
 
 
   loadClassroomIfNecessary: ->
-    if not @classroomId and not @courseInstanceId
-      @onAccessibleLevelLoaded()
-      return
     if @headless and not @level?.isType('web-dev')
       @onAccessibleLevelLoaded()
       return
+
+    if not @classroomId and not @courseInstanceID
```

Comment:
@smallst ,  it was `@courseInstanceId` before which is not the actual variable name causing the level to load when course-instance id is not passed. 🤦

Reply:
omg..